Lib/fontTools/varLib/models.py

```diff
@@ -425,24 +425,29 @@ def piecewiseLinearMap(v, mapping):
 def main(args):
 	"""Normalize locations on a given designspace"""
 	from fontTools import configLogger
+	import argparse
 
-	args = args[1:]
+	parser = argparse.ArgumentParser(
+		"fonttools varLib.models",
+		description=main.__doc__,
+	)
+	parser.add_argument('-l', '--loglevel', metavar='LEVEL', default="INFO",
+		help="Logging level (defaults to INFO)")
 
-	# TODO: allow user to configure logging via command-line options
-	configLogger(level="INFO")
+	group = parser.add_mutually_exclusive_group(required=True)
+	group.add_argument('--designspacefile',metavar="DESIGNSPACE",type=str)
```

Comment:
`--designspacefile` is too verbose, `--designspace` is enough.
maybe also add `-d` short option (and `-l` for the `--locations`)